etc/icinga2/scripts/mail-host-notification.sh

```diff
@@ -129,6 +131,12 @@ if [ -n "$HOSTADDRESS6" ] ; then
 IPv6:	 $HOSTADDRESS6"
 fi
 
+## Check whether host_notes was specified.
```

Comment:
I'd write host_notes more human-readable -like IPv4 above- i.e. `host notes`.

Reply:
Also no problem, I wanted to submit the variables as specific as possible, so host_notes or host.notes was best imho.